Robot2019/src/main/java/frc/robot/commands/Climb.java

```diff
@@ -10,19 +10,24 @@
 import edu.wpi.first.wpilibj.command.Command;
 import frc.robot.subsystems.Climber;
 import frc.robot.subsystems.Drivetrain;
+import edu.wpi.first.wpilibj.Joystick;
 
 public class Climb extends Command {
   private Climber climber;
   private Drivetrain dt;
+  private Joystick dtJoy;
   private boolean up;
+  private State state;
 
-  public Climb(Climber climber, Drivetrain dt) {
+  public Climb(Climber climber, Drivetrain dt, Joystick joy) {
     // Use requires() here to declare subsystem dependencies
     this.climber = climber;
     this.dt = dt;
+    dtJoy = joy;
     requires(climber);
     requires(dt);
     up = true;
```

Comment:
It does not seem like the variable "up" is ever being used in Climb.java. Is this variable supposed to be replaced with the state machine?

Reply:
up is used in climb()

Reply:
My bad, I did not see that.